packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx

```diff
@@ -155,7 +155,7 @@ export const MetricsTable = () => {
         ),
         state: {
             sorting,
-            showProgressBars: isFetching,
+            isLoading: isFetching,
```

Comment:
on the main ticket it said to use `violet` could you pass this color to the loader props ?